internal/helm/helmchart_structure_exists.go

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package helm
+
+import (
+	"github.com/eclipse-tractusx/tractusx-quality-checks/internal"
+	"github.com/eclipse-tractusx/tractusx-quality-checks/internal/repo"
+	"os"
+	"strings"
+)
+
+type HelmStructureExists struct {
+}
+
+// NewHelmStructureExists returns a new check based on QualityGuideline interface.
+func NewHelmStructureExists() txqualitychecks.QualityGuideline {
+	return &HelmStructureExists{}
+}
+
+func (r *HelmStructureExists) Name() string {
+	return "TRG 5.02 - Chart structure"
+}
+
+func (r *HelmStructureExists) Description() string {
+	return "Helm Chart should follow a specific structure."
+}
+
+func (r *HelmStructureExists) ExternalDescription() string {
+	return "https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-02"
+}
+
+func (r *HelmStructureExists) Test() *txqualitychecks.QualityResult {
+
```

Comment:
I see this on multiple of your files. The empty blank lines at the beginning of the function seems unnecessary for me.